src/contexts/bridge.context.tsx

```diff
@@ -136,8 +137,8 @@ const BridgeProvider: FC<PropsWithChildren> = (props) => {
     []
   );
 
-  const estimateBridgeGasPrice = useCallback(
-    ({ from, to, token, destinationAddress }: EstimateBridgeGasPriceParams) => {
+  const estimateBridgeGasLimit = useCallback(
+    ({ from, to, token, destinationAddress }: EstimateBridgeFee) => {
```

Comment:
Looks like this function `estimateBridgeGasLimit` is the one that should be named `estimateBridgeFee` since it's the one returning the interface `EstimateBridgeFee`.

Can we rename the current function `estimateBridgeFee` returning a BigNumber to something else?